scalalib/src/mill/scalalib/SemanticDbJavaModule.scala

```diff
@@ -21,6 +21,16 @@ trait SemanticDbJavaModule extends JavaModule { hostModule =>
     ).asInstanceOf[String]
   }
 
+  def semanticDbJavaVersion: Input[String] = T.input {
+    T.env.getOrElse(
+      "JAVASEMANTICDB_VERSION",
+      SemanticDbJavaModule.contextJavaSemanticDbVersion.get()
+        .getOrElse(
+          SemanticDbJavaModule.buildTimeJavaSemanticDbVersion
+        )
+    ).asInstanceOf[String]
```

Comment:
```suggestion
    T.env.getOrElse(
      "JAVASEMANTICDB_VERSION",
      SemanticDbJavaModule.contextJavaSemanticDbVersion.get()
        .getOrElse[String](
          SemanticDbJavaModule.buildTimeJavaSemanticDbVersion
        )
    }
```

Reply:
This one does not work, but I already found a nicer version.